### ISSUE TITLE
Use Google Mirror Maven Central based on the region

### ISSUE THREAD

**Issue by jmatsu** [DroidKaigi/conference-app-2019#361]
## Overview (Required)

- Google Mirror Maven Central could reduce the sync time (see https://github.com/DroidKaigi/conference-app-2019/issues/168) though, CI region might be unknown so we cannot use suitable ones. 
- Now we might be able to use ipinfo.io and country.to for this stuff. https://ipinfo.io/country can return the country of the caller and http://country.io/continent.json is an index of regions by countries.
- In short, we may be able to select a suitable mirror somehow.

## Links
- Related to https://github.com/DroidKaigi/conference-app-2019/issues/168
